compendium-browser.js

```diff
@@ -1219,6 +1219,7 @@ class CompendiumBrowser extends Application {
                         return false;
                     }
                 } else {
+                    if (prop === undefined) return false;

```

Comment:
This line is my only concern, as it does change behavior for some filters, but it seemed to be an oversight anyway